config/overlays/e2e/kustomization.yaml

```diff
@@ -0,0 +1,38 @@
+resources:
```

Comment:
I'd want to avoid a separate overlay for e2e tests, I think we should aim to combine this with the `overlays/dev` directory since they have the same purpose

Reply:
with latest commit I completely removed the `e2e` overlay and I started relying on the `dev` one.